schema/swagger.yml

```diff
@@ -123,6 +123,7 @@ paths:
           description: Address of account
           type: string
           format: address
+          pattern: '^[0-9]+L$'
```

Comment:
We shouldn't allow addresses that starts from `0`

Reply:
That would be: `^[1-9][0-9]*L$` :wink:

Reply:
Guys you're right but I've uses the same one we have in our z_schema: https://github.com/LiskHQ/lisk/blob/960d316c177f8c300d89c7df01f6ed2911ab9aba/helpers/z_schema.js#L88
We can tackle it in another issue to keep the scope of this PR. What do you think @karmacoma ?

Another point to highlight is the reason I have introduced this pattern attribute: `format: address` line presented several times in Swagger specs is useless. Correct me if I am wrong @nazarhussain

Reply:
I think we should fix it as part of current issue.

Reply:
I think out of scope of this particular issue, and that we should address separately.